api/all/src/main/java/io/opentelemetry/api/internal/ImmutableKeyValuePairs.java

```diff
@@ -29,9 +29,7 @@
 @Immutable
 public abstract class ImmutableKeyValuePairs<K, V> {
 
-  protected List<Object> data() {
-    return Collections.emptyList();
```

Comment:
This was only needed because `ImmutableBaggage` wasn't abstract which it should be